src/cascadia/TerminalApp/CascadiaSettings.cpp

```diff
@@ -673,3 +688,26 @@ void CascadiaSettings::_ValidateKeybindings()
         _warnings.insert(_warnings.end(), keybindingWarnings.begin(), keybindingWarnings.end());
     }
 }
+
+std::string CascadiaSettings::_ApplyFirstRunChangesToSettingsTemplate(std::string_view settingsTemplate) const
+{
+    std::string finalSettings{ settingsTemplate };
+    auto replace{ [](std::string& haystack, std::string_view needle, std::string_view replacement) {
+        auto pos{ std::string::npos };
+        while ((pos = haystack.rfind(needle, pos)) != std::string::npos)
+        {
+            haystack.replace(pos, needle.size(), replacement);
+        }
+    } };

```

Comment:
I wonder how many projects have had to write this exact thing.

Reply:
:laughing_but_crying_inside: